diff.py

```diff
@@ -1899,6 +1947,35 @@ def process(dump: str, config: Config) -> List[Line]:
     return output
 
 
+# Post-process the asm lines in-place. Includes:
+#  - De-referencing relocations
+def post_process(lines: List[Line]):
+    post_process_relocations(lines)
+
+def post_process_relocations(lines: List[Line]):
+    for line in lines:
+        reloc_match = re.search(RELOC_PLACEHOLDER_PATTERN, line.original)
+        if reloc_match is None:
+            continue
+
+        # Get value at relocation
+        reloc = reloc_match.group(0)
+        line_number = re.search(RELOC_PLACEHOLDER_PATTERN, reloc).group(1)
+        line_original = get_line(lines, int(line_number, 16)).original
+        value = line_original.split()[1]
+
+        # Replace relocation placeholder with value
+        replaced = re.sub(RELOC_PLACEHOLDER_PATTERN, 
+                          f"={value} ({line_number})", line.original)
+        line.original = replaced
+
+def get_line(lines: List[Line], line_number: int) -> Optional[Line]:
```

Comment:
inline this within post_process_relocations, and make it use a lookup table so it's not O(n^2)

Reply:
Done.